libvcx/src/api_lib/api_c/issuer_credential.rs

```diff
@@ -886,23 +825,18 @@ pub extern "C" fn vcx_issuer_revoke_credential_local(
     check_useful_c_callback!(cb, VcxErrorKind::InvalidOption);
 
     let source_id = issuer_credential::get_source_id(credential_handle).unwrap_or_default();
-    let issuer_did: String = match settings::get_config_value(settings::CONFIG_INSTITUTION_DID) {
-        Ok(err) => err,
-        Err(err) => return err.into(),
-    };
     info!(
-        "vcx_issuer_revoke_local(command_handle: {}, credential_handle: {}, issuer_did: {}) source_id: {}",
-        command_handle, credential_handle, issuer_did, source_id
+        "vcx_issuer_revoke_local(command_handle: {}, credential_handle: {}) source_id: {}",
```

Comment:
The function is called `vcx_issuer_revoke_credential_local` (same name makes it easier for fulltext search it based on logs)